cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -122,6 +133,16 @@ func existingConversionFunctions(context *generator.Context) conversions {
 	return preexisting
 }
 
+func isAPIPackage(path string) bool {
+	apiPackages := []string{"k8s.io/kubernetes/pkg/api/", "k8s.io/kubernetes/pkg/apis/"}
```

Comment:
I agree it's definitely not idea.

Regarding your idea:
1. it would have to be "/apis/" or "/api/"
2. I'm not sure if we can check if it has types.go file - can we easily check it.